### DISCUSSION TITLE
Startup time

### DISCUSSION THREAD

**Discussion by fainley** [sqlitebrowser/sqlitebrowser#2875] [Q&A]
I don't have a slow PC but DB Browser takes about 15 seconds to start without opening any databases. I was wondering what it was doing and how I could speed it up.

**Comment by chrisjlocke** [comment]
This could be the check for new versions?  It's the only thing that normally occurs at startup.
In Edit, Preferences, turn off 'automatic check for updates' and try again?

What's your OS?  Whhat version have you installed?

**Comment by fainley** [comment]
Hi Chris,

No, automatic updates is unchecked. I'm using Windows 7 Pro 64 bit. Typically, I just closed Db4s down and restarted it and it only took about 3 seconds. However, that is the fastest it's ever started. It really does normally take 10-15  seconds. I'll try it on Windows 10 in the next few days.

Thank for your help.

Francis

**Reply by chrisjlocke** [reply]
On my Win 10 laptop, it can take ~4 seconds to start (on the copy I just fired up).  Are you using the normal version or the SQLCipher version?
